modules/oadp-creating-backup-cr.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-creating-backup-cr_{context}"]
+= Creating a Backup CR
+
+You back up Kubernetes images, internal images, and persistent volumes (PVs) by creating a `Backup` custom resource (CR).
+
+.Prerequisites
+
+* You must install the OpenShift API for Data Protection (OADP) Operator.
+* The `DataProtectionApplication` CR must be in a `Ready` state.
+* Backup prerequisites:
+** You must have S3 object storage configured for Velero.
+** You must have a backup location configured in the `DataProtectionApplication` CR.
+* Snapshot prerequisites:
+** You must have a cloud provider that uses a native snapshot API or the Container Snapshot Interface (CSI) to create snapshots, such as OpenShift Container Storage 4.
```

Comment:
Not clear to me. 
_You must have a cloud provider that uses a native snapshot API or a cloud provider that uses the Container Snapshot Interface (CSI) to create snapshots...._
 -or-
 _You must have a cloud provider that uses a native snapshot API or you must have the Container Snapshot Interface (CSI) to create snapshots...._
 ??